### ISSUE TITLE
[AdminListBundle] invalid AbstractAdminListConfigurator getAddUrlFor() for menus

### ISSUE THREAD

**Issue by jerome2710** [Kunstmaan/KunstmaanBundlesCMS#3450]
When managing menus in Kunstmaan 7.1, the `add["path"]` is wrong, resulting in a non-existent route:

```
An exception has been thrown during the rendering of a template ("None of the chained routers were able to generate route: Route 'kunstmaan\menubundle_admin_menuitem_add' not found").
```

This originates from [getAddUrlFor()](https://github.com/Kunstmaan/KunstmaanBundlesCMS/blob/7.1/src/Kunstmaan/AdminListBundle/AdminList/Configurator/AbstractAdminListConfigurator.php#L186), which calls [getPathByConvention()](https://github.com/Kunstmaan/KunstmaanBundlesCMS/blob/7.1/src/Kunstmaan/AdminListBundle/AdminList/Configurator/AbstractAdminListConfigurator.php#L815).

The `$rootNamespace` contains `Kunstmaan\MenuBundle` and therefore creates a wrong path. We have temporarily added the following patch on [AbstractAdminListConfigurator.php#L818](https://github.com/Kunstmaan/KunstmaanBundlesCMS/blob/7.1/src/Kunstmaan/AdminListBundle/AdminList/Configurator/AbstractAdminListConfigurator.php#L818) to remove the `\`:

```php
$rootNamespace = str_replace('\\', '', $rootNamespace);
```

**Comment by dannyvw** [comment]
Same as https://github.com/Kunstmaan/KunstmaanBundlesCMS/pull/3420/files? This is not released yet.

**Comment by jerome2710** [comment]
@dannyvw it might be related, but this is pushed into [Kunstmaan:6.4](https://github.com/Kunstmaan/KunstmaanBundlesCMS/tree/6.4) and we are using 7.1.

**Comment by dannyvw** [comment]
@jerome2710 That change is not yet released on a 7.1 tag. It exists in the 7.1 branch.

**Comment by acrobat** [comment]
https://github.com/Kunstmaan/KunstmaanBundlesCMS/releases/tag/7.1.2 was released!
